semgrep-core/src/analyzing/controlflow_build.ml

```diff
@@ -692,7 +698,7 @@ let (control_flow_graph_of_stmts : parameter list -> stmt list -> F.flow) =
 
   let state =
     {
-      g;
+      g = { graph = g; entry = enteri };
```

Comment:
Again, this causes a cascade of changes, and I am not sure it's justified unless we need the `entry` somewhere in the construction of the graph.